quickstarts/03-durable-agent-tool-call/durable_weather_agent.py

```diff
@@ -0,0 +1,34 @@
+from dapr_agents import DurableAgent
+from dotenv import load_dotenv
+from weather_tools import tools
+import asyncio
+import logging
+
+
+async def main():
+    load_dotenv()
+    logging.basicConfig(level=logging.INFO)
+
+    # 1️⃣ Instantiate your agent (no .as_service())
+    weather_agent = DurableAgent(
+        role="Weather Assistant",
+        name="Stevie",
+        goal="Help humans get weather and location info using smart tools.",
+        instructions=[
+            "Respond clearly and helpfully to weather-related questions.",
+            "Use tools when appropriate to fetch weather data.",
+        ],
+        message_bus_name="messagepubsub",
+        state_store_name="workflowstatestore",
+        state_key="workflow_state",
+        agents_registry_store_name="agentstatestore",
+        agents_registry_key="agents_registry",
+        broadcast_topic_name=None,
+        tools=tools,
+    )
+    # 2️⃣ Start the agent service
+    await weather_agent.run("What's the weather in Boston?")
```

Comment:
so durable agent can now run with the run()? Can it still do the .as_service() for a long lived agent? I can't remember if we have it as a starting mechanism or not, but would be nice to also have an as_consumer() where a pubsub event could trigger its start

Reply:
if it can support both, can you update the comment pls to say it could be optional to instead use the .as_service? or does the run() enable both means of start?

Reply:
oh yes we can run it with as service, etc. Everything that already existed still works. I just wanted to work on your comment from a previous PR. You mentioned that it would be good to be able to also have a RUN command :)